src/card.css

```diff
@@ -0,0 +1,139 @@
+/* stylelint-disable string-quotes */
+/* stylelint-disable color-hex-length */
+/* stylelint-disable declaration-block-trailing-semicolon */
+/* stylelint-disable shorthand-property-no-redundant-values */
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap');
+
+.product-card {
+  display: block;
+  background-position: center;
+  height: 406px;
+  width: 198px;
+  margin: 0 auto;
+  border: 1px solid #f3f3f3;
+  justify-content: center;
+  align-items: center;
+
+}
+
+.product-image {
+  position: flex;
+  background-image: url("images/imac.jpeg");
+  background-size: cover;
+  background-position: center;
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px 19px;
+}
+
+.product-image img {
+  max-width: 100%;
+  max-height: 100%;
+  display: block;
+}
+
+.product-name {
+  width: 166px;
+  height: 36px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  bottom: 165px;
+  line-height: 18px;
+  color: #060b35;
+  margin: -4px 16px 4px 16px;
+}
+
+.product-code {
+  display: box;
+  width: 98px;
+  height: 14px;
+  bottom: 147px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 4px 84px 16px 16px;
```

Comment:
Add margins or paddings to whole your card, no need to set it in each element

fix everywhere